### PR TITLE
ADF-2154 remove references to free plan on our

### PR DESCRIPTION
**Pull request by vprigent**

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -206,7 +204,7 @@ The AddressFinder website has a large [FAQ section](https://addressfinder.com.au
 = 1.2.12 =
 * Fixes errors when checkouts are missing address_line_2 field
 = 1.2.11 =
-* Checks window.AddressFinder is loaded before intialising plugin
+* Checks window.Addressfinder is loaded before intialising plugin
```
Comment (santoshm) [inline readme.txt:207]: If this is referencing what the actual source is doing, then I am not certain the source has changed the case from `AddressFinder` to `Addressfinder` 🤷

**Review by santoshm — APPROVED** [review]
LGTM. Left a few small questions for my clarification.

File: DEVELOPMENT.md [hunk 2]
```diff
@@ -1,107 +0,0 @@
-# Development
```
[the remark this answers is not in the document]
Comment (vprigent) [reply DEVELOPMENT.md:1]: This repo is public and this file contains information on how to access the dev environment for AF/Abletech. While it doesn't contain the password associated with the dev env, its still a potential entry door for attackers.

[the remark this answers is not in the document]
Comment (vprigent) [reply DEVELOPMENT.md:1]: Its also been moved to our internal wiki so its not lost.

File: readme.md [hunk 3]
```diff
@@ -1,18 +1,18 @@
...
 
-Existing users can obtain their API key from the [AddressFinder Portal](https://portal.addressfinder.net/?utm_source=woocommerce%20plugin&utm_medium=plugin&utm_campaign=plugin&utm_term=AddressFinder%20Portal&utm_content=existing%20users).
+Existing users can obtain their API key from the [Addressfinder Portal](https://portal.addressfinder.net/?utm_source=woocommerce%20plugin&utm_medium=plugin&utm_campaign=plugin&utm_term=Addressfinder%20Portal&utm_content=existing%20users).
 
-Installation instructions and more information on the AddressFinder [WooCommerce Plugin](https://addressfinder.nz/docs/woocommerce?utm_source=woocommerce%20plugin&utm_medium=plugin&utm_campaign=plugin&utm_term=WooCommerce%20Plugin&utm_content=Read%20More) page.
+Installation instructions and more information on the Addressfinder [WooCommerce Plugin](https://addressfinder.nz/docs/woocommerce?utm_source=woocommerce%20plugin&utm_medium=plugin&utm_campaign=plugin&utm_term=WooCommerce%20Plugin&utm_content=Read%20More) page.
```
[the remark this answers is not in the document]
Comment (vprigent) [reply readme.md]: Fixed that one btw.

**Pull request merged**